subsys/modem/modem_cmux.c

```diff
@@ -23,7 +23,7 @@ LOG_MODULE_REGISTER(modem_cmux, CONFIG_MODEM_MODULES_LOG_LEVEL);
 #define MODEM_CMUX_DATA_FRAME_SIZE_MIN		(MODEM_CMUX_FRAME_SIZE_MAX + \
 						 MODEM_CMUX_DATA_SIZE_MIN)
 
-#define MODEM_CMUX_CMD_DATA_SIZE_MAX		(0x04)
+#define MODEM_CMUX_CMD_DATA_SIZE_MAX		(0x08)
```

Comment:
Confirmed as working. I had initially tested with 0x10 but 0x08 seems to be enough to get rid of the buffer overrun I was seeing otherwise.

Reply:
Nice :) I replicated the exact data that was sent from your modem in the unit test which is why I decided to go with a smaller buffer with confidence :)